public/app/screens/main/navigation/navigation.jsx

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var React = require('react'),
+  Link = require('react-router').Link,
+  StyleMixin = require('mixins/style-mixin');
+
+var Navigation = React.createClass({
+  mixins: [StyleMixin(require("./style.scss"))],
+  render: function () {
+    // TODO: Use mixins instead of CSS classes
+    // TODO: Pass number of elements to mixin instead of specifying class name
+    var classes = "cc_navigation icon-bar two-up " + this.props.orientation;
+    
+    var items = this.props.items.map(function (item, i) {
+      return (
+        <Link to={item.linkTo} key={i} className="cc_navigation__link item">
```

Comment:
@larsblumberg two comments:
1. `item` should be fixed in a separate scope, as it is not a part of the template
2. `key` is not needed, if you are ok with the array indices used as keys. React do that automatically (https://github.com/facebook/react/issues/1342#issuecomment-39274004)